### PR TITLE
[irods/irods#2759] Documented <PRIORITY> delay rule hint (4-2-stable)

### PR DESCRIPTION
**Pull request by korydraughn**
Related to irods/irods#2759

### CODE REVIEW

File: docs/plugins/pluggable_rule_engine.md [hunk 1]
```diff
@@ -187,24 +187,29 @@ delay("hints") {
     - `dd.hh:mm:ss` - 2-digit integers representing days, hours, minutes, and seconds, respectively. Most significant values can be omitted (e.g. 20:40 means mm:ss)
     - `YYYY-MM-DD.hh:mm:ss` - Least significant values can be omitted (e.g. 2015-07-29.12 means noon of July 29, 2015)
   - `PLUSET` - Relative Execution Time - Relative to current time when the delayed execution should be performed.
+  - `PRIORITY` - Scheduled Execution Order - Controls the order at which rules are scheduled for execution.
+  - `PRIORITY` - Scheduled Execution Order - Controls the execution order of scheduled rules.
+  - `PRIORITY` - Scheduled Execution Order - Controls when the rule will be scheduled for execution in relation to sibling rules.
+  - `PRIORITY` - Scheduled Execution Order - A hint to the delay server on how to schedule rules for execution.
```
Comment (korydraughn) [inline docs/plugins/pluggable_rule_engine.md:193]: Which one do we want to use?

File: docs/plugins/pluggable_rule_engine.md [hunk 2]
```diff
@@ -187,24 +187,29 @@ delay("hints") {
...
+    - A valid priority value is an integer in the range [1, 9] where 1 represents the lowest level and 9 represents the highest level.
```
Comment (korydraughn) [inline docs/plugins/pluggable_rule_engine.md:194]: This would be paired with one of the statements above.

Comment (trel) [reply docs/plugins/pluggable_rule_engine.md:194]: I like the first...  building on it...

```
  - `PRIORITY` - Scheduled Execution Priority - Controls the execution order of scheduled rules. Valid priority values are integers in the range [1, 9] where 9 represents the highest priority (executed earlier).  Default priority is 5.
```

Comment (korydraughn) [reply docs/plugins/pluggable_rule_engine.md:194]: Ok. I'll add those changes and let you see how it is rendered.

Comment (korydraughn) [reply docs/plugins/pluggable_rule_engine.md:194]: Did you see the very first PRIORITY statement? This code view doesn't show all of the choices.

Comment (trel) [reply docs/plugins/pluggable_rule_engine.md:194]: yes, i saw all four.

File: docs/plugins/pluggable_rule_engine.md [hunk 3]
```diff
@@ -192,6 +192,7 @@ delay("hints") {
   - `PRIORITY` - Scheduled Execution Order - Controls when the rule will be scheduled for execution in relation to sibling rules.
   - `PRIORITY` - Scheduled Execution Order - A hint to the delay server on how to schedule rules for execution.
     - A valid priority value is an integer in the range [1, 9] where 1 represents the lowest level and 9 represents the highest level.
+  - `PRIORITY` - Scheduled Execution Order - Controls the execution order of scheduled rules. Valid priority values are integers in the range [1, 9] where 9 represents the highest priority (executed earlier). Defaults priority is 5.
```
Comment (trel) [inline docs/plugins/pluggable_rule_engine.md:195]: Defaults -> Default

Comment (korydraughn) [reply docs/plugins/pluggable_rule_engine.md]: done

File: docs/plugins/pluggable_rule_engine.md (hunk 2 above)
Comment (korydraughn) [reply docs/plugins/pluggable_rule_engine.md:194]: Ok. Preparing to update and add the #

Comment (korydraughn) [reply docs/plugins/pluggable_rule_engine.md]: done

**Comment by korydraughn** [comment]
added #

**Pull request merged**